src/main/java/blarb/Processor.java

```diff
@@ -0,0 +1,145 @@
+                outputs[0] = output;
+                storage.file(task);
+            } catch (IOException ex) {
+                outputs[1] = "Cannot access storage.";
+            } catch (InputMismatchException ex) {
+                outputs[0] = "Todo what?";
+            }
+            break;
+        case DEADLINE:
+            try {
+                if (cml.isSingleCommand()) {
+                    throw new InputMismatchException();
+                }
+                String[] fragments = cml.description.split(" /by ");
+                Task task = new Deadline(fragments[0], fragments[1]);
+                String output = tasklist.add(task);
+                outputs[0] = output;
+                storage.file(task);
+            } catch (IOException ex) {
+                outputs[1] = "Cannot access storage.";
+            } catch (ArrayIndexOutOfBoundsException ex) {
+                outputs[0] = "Type the deadline, then give the time using \"/by\".";
+            } catch (DateTimeParseException ex) {
+                outputs[0] = "blarb.Deadline time must be in the format of yyyy-mm-dd.";
+            } catch (InputMismatchException ex) {
+                outputs[0] = "Someone's having trouble with deadlines.";
+            }
+            break;
+        case EVENT:
+            try {
+                if (cml.isSingleCommand()) {
+                    throw new InputMismatchException();
+                }
+                String[] fragments = cml.description.split(" /at ");
+                Task task = new Event(fragments[0], fragments[1]);
+                String output = tasklist.add(task);
+                outputs[0] = output;
+                storage.file(task);
+            } catch (IOException ex) {
+                outputs[1] = "Cannot access storage.";
+
+            } catch (ArrayIndexOutOfBoundsException ex) {
+                outputs[0] = "Type the event, then give the time using \"/at\".";
+            } catch (InputMismatchException ex) {
+                outputs[0] = "Tell me the event!";
+            }
+            break;
+        case FIND:
+            try {
+                if (cml.isSingleCommand()) {
+                    throw new InputMismatchException();
+                }
+                outputs[0] = tasklist.find(cml.description);
+            } catch (InputMismatchException ex) {
+                outputs[0] = "I need a clue to find stuff!";
+            }
+            break;
+        case LIST:
+            outputs[0] = cml.isSingleCommand()
+                    ? tasklist.list()
+                    : "Listing is simple, so typing \"list\" would suffice.";
+            break;
+        case BYE:
+            outputs[0] = cml.isSingleCommand()
+                    ? "Hasta la vista, baby."
+                    : "Type \"bye\" to see me go.";
+            break;
+        default:
+            outputs[0] = String.format("I have detailed files on human anatomy, but not %s.", input);
```

Comment:
Indentation for wrapped lines should be 8 spaces (i.e. twice the normal indentation of 4 spaces) more than the parent line